testlib/dbus.py

```diff
@@ -407,17 +407,12 @@ def pool_set_param(pool_path, param_iface, dbus_param, dbus_value):
             dbus.PROPERTIES_IFACE,
         )
 
-        try:
-            iface.Set(
-                param_iface,
-                dbus_param,
-                dbus_value,
-                timeout=StratisDbus._TIMEOUT,
-            )
-        except dbus.exceptions.DBusException:
-            return False
-
-        return True
+        return iface.Set(
```

Comment:
Document what the method returns in the docs. You could also document that it raise dbus.exceptions.DBusException.